hr_timesheet_sheet/models/hr_timesheet_sheet.py

```diff
@@ -458,33 +458,30 @@ def _get_default_analytic_line(self, date, project, task, timesheet=None):
                 'sheet_id': self.id,
             })
         if timesheet:
-            amount = sum([t.unit_amount for t in timesheet])
+            unit_amount = sum([t.unit_amount for t in timesheet])
             values.update({
-                'unit_amount': amount,
+                'unit_amount': unit_amount,
             })
         return values
 
     @api.model
-    def _prepare_empty_analytic_line(self):
+    def _prepare_empty_analytic_line(self, project, task):
         return {
             'name': '/',
             'employee_id': self.employee_id.id,
             'date': self.date_start,
-            'project_id': self.add_line_project_id and
-            self.add_line_project_id.id or False,
-            'task_id': self.add_line_task_id and
-            self.add_line_task_id.id or False,
+            'project_id': project and project.id,
```

Comment:
won't project.id resolve to false if project is empty?

Reply:
same answer as above